lua/resession/util.lua

```diff
@@ -168,4 +168,18 @@ M.shorten_path = function(path)
   end
 end
 
+--- Trigger a resession `User` event
+---@param event string The event name to be appended to "Resession"
+---@param delay? boolean Whether or not to delay the event asynchronously (Default: true)
+M.event = function(event, delay)
+  local emit_event = function()
+    vim.api.nvim_exec_autocmds("User", { pattern = "Resession" .. event, modeline = false })
+  end
+  if delay == false then
+    emit_event()
+  else
+    vim.schedule(emit_event)
+  end
```

Comment:
The only callsite now doesn't pass in `delay = false`, so I think we can just remove this parameter

Reply:
>I think this got mistakenly resolved

Oh, I just assumed it wouldn't make sense anymore for grepping since we now map hooks directly to event names.